doc/sphinx/cti/phases.rst

```diff
@@ -96,7 +96,8 @@ example, if a phase definition specifies the elements as::
 then when this definition is imported by an application, element-specific
 properties will be ordered in the same way::
 
-    >>> gas = importPhase('example.cti', 'gasmix')
```

Comment:
Good catch on this one, given that the Python function `importPhase` hasn't even existed since the legacy Python module went away.